fix: rendering for active/comoute hours rendering (#1464)
neondatabase/website: content/docs/introduction/how-billing-works.md

These allowances are included in your plan's monthly fee, except for the Free Tier, which is always free. You can find the monthly fees for the [Launch](/docs/introduction/plans#launch) and [Scale](/docs/introduction/plans#scale) plans on our [pricing](https://neon.tech/pricing) page. The [Enterprise](/docs/introduction/plans#enterprise) plan is completely customizable with respect to allowances.

<Admonition type="tip" title="What is a compute hour?">

> **Active hours** meausre the amount of time a compute is active as opposed to being idle due to being suspended due to inactivity. The time that your compute is idle is not counted toward compute usage.

Reviewer: ```suggestion
**Active hours** measures the amount of time a compute is active, rather than idle when suspended due to inactivity. The time that your compute is idle is not counted toward compute usage.
```
